src/main/resources/db/liq/changelog.xml

```diff
@@ -0,0 +1,163 @@
+                                  baseTableName="categories"
+                                  constraintName="fk_categories_languages"
+                                  referencedColumnNames="language_id"
+                                  referencedTableName="languages"/>
+    </changeSet>
+    <!--> <-->
+    <changeSet id="5" author="aleksdenni">
+        <createTable tableName="questions">
+            <column name="question_id" type="int" autoIncrement="true">
+                <constraints primaryKey="true" nullable="false"/>
+            </column>
+            <column name="question" type="varchar(255)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="category_id" type="int">
+                <constraints nullable="false"/>
+            </column>
+        </createTable>
+    </changeSet>
+    <!--> <-->
+    <changeSet  author="aleksdenni"  id="6">
+        <addForeignKeyConstraint  baseColumnNames="category_id"
+                                  baseTableName="questions"
+                                  constraintName="fk_questions_categories"
+                                  referencedColumnNames="category_id"
+                                  referencedTableName="categories"/>
+    </changeSet>
+    <!--> answer ? structure <-->
+    <changeSet id="7" author="aleksdenni">
+        <createTable tableName="options">
+            <column name="option_id" type="int" autoIncrement="true">
+                <constraints primaryKey="true" nullable="false"/>
+            </column>
+            <column name="answer" type="varchar(255)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="question_id" type="int">
+                <constraints nullable="false"/>
+            </column>
+            <column name="correct" type="bool">
+                <constraints nullable="false"/>
+            </column>
+        </createTable>
+    </changeSet>
+    <!--> <-->
+    <changeSet  author="aleksdenni"  id="8">
+        <addForeignKeyConstraint  baseColumnNames="question_id"
+                                  baseTableName="options"
+                                  constraintName="fk_options_questions"
+                                  referencedColumnNames="question_id"
+                                  referencedTableName="questions"/>
+    </changeSet>
+    <!--> successful? name column? <-->
+    <changeSet id="9" author="aleksdenni">
+        <createTable tableName="passed_test">
+            <column name="passed_test_id" type="int" autoIncrement="true">
+                <constraints primaryKey="true" nullable="false"/>
+            </column>
+            <column name="user_id" type="int">
+                <constraints nullable="false"/>
+            </column>
+            <column name="category_id" type="int">
+                <constraints nullable="false"/>
+            </column>
+            <column name="timestamp" type="int">
```

Comment:
Why type is int?
There are dedicated types for dates in PostreSQL: 
https://www.postgresql.org/docs/current/datatype-datetime.html 
E.g., timestamp)))